qa/zcash/performance-measurements.sh

```diff
@@ -357,9 +357,9 @@ case "$1" in
                 cat valgrind.out
                 rm -f valgrind.out
                 ;;
-            test_bitcoin)
+            test_zcash)
                 rm -f valgrind.out
```

Comment:
Here we have a problem. This pr is not changing the boost test binary(located at `zcash/src/test`) and with name `test_bitcoin`.

We could change the name of the binary however i am not totally sure how much is involved. For sure we need to change https://github.com/zcash/zcash/blob/master/src/test/Makefile and maybe https://github.com/zcash/zcash/blob/master/Makefile.am too.

Reply:
Thanks for catching this, I tinkered with changing that name, but I think we should defer changes outside qa...